detox/test/e2e/copilot/07.copilot.assertions.test.js

```diff
@@ -0,0 +1,143 @@
+      ).rejects.toThrowError();
+    });
+
+    it('should assert an element is visible (by id)', async () => {
+      await copilot.perform(
+        'Verify there is an element with ID "subtext-root"'
+      );
+    });
+
+    it('should not assert an element that does not exist (by id)', async () => {
+      await jestExpect(async () =>
+        await copilot.perform('Find an element with ID "RandomJunk959" in the screen')
+      ).rejects.toThrowError();
+    });
+
+    it('should assert an element has (accessibility) label', async () => {
+      await copilot.perform(
+        'Verify there is an element with the accessibility label "I contain some text"'
+      );
+    });
+
+    it('should not assert an element that does not exist (by label)', async () => {
+      await jestExpect(async () =>
+        await copilot.perform('Find an element with label "I exist in the screen" in the screen')
+      ).rejects.toThrowError();
+    });
+
+    it('assert toggle exist by element type', async () => {
+      await copilot.perform(
+        'Verify that the toggle type element is exist in the screen',
+      );
+    });
+
+    //Note: This test is skipped because currently we don't know what are the exact expectations.
+    //Should we expect the copilot to understand the similarity between toggle and checkbox or not?
+    it.skip('shouldn`t assert an element that does not exist (by type)', async () => {
+      await jestExpect(async () =>
+        await copilot.perform('Find a Check-box type element in the screen')
+      ).rejects.toThrowError();
+    });
+
+    it('shouldn`t assert an element that does not exist (by type, text-field)', async () => {
+      await jestExpect(async () =>
+        await copilot.perform('Find a text-field element in the screen')
+      ).rejects.toThrowError();
+    });
+
+    it('assert toggle-switch widget false', async () => {
+      await copilot.perform(
+        'Verify that the toggle has false value',
+      );
+
+      await expect(element(by.id('toggle'))).toHaveToggleValue(false);
+    });
+
+    it('assert toggle-switch widget true ', async () => {
+      await copilot.perform(
+        'Tap the toggle',
+        'Verify that the toggle has true value'
+      );
+
+      await expect(element(by.id('toggle'))).toHaveToggleValue(true);
+    });
+
+    it('assert details in image', async () => {
+      await copilot.perform(
+        'Verify there is an image element in the screen',
+        'Verify there is a banana in the image element',
+        'Verify there isn`t an apple in the image element',
+        'Verify there are grapes in the image element',
+      );
+    });
+
+    it('assert element`s color', async () => {
+      await copilot.perform(
+        'Verify there is a green text element'
+      );
+    });
+
+    //Note: When removing "under the image element" the test failed, saying there is no blue text element in the screen, only pink and black.
```

Comment:
the convention is to use space after comment start (`//Something` → `// Something`)